style.css

```diff
@@ -5,33 +5,41 @@
 		Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
 }
 
-.green {
-	color: green;
-}
+/* MOBILE */
+@media (max-width: 600px) {
```

Comment:
Why do we use max-width here, can we go for min-width: 200px for example to follow the mobile first approach?